include/filters/increment.hpp

```diff
@@ -155,7 +157,7 @@ bool MultiChannelIncrementFilter<T>::update(const std::vector<T> & data_in, std:
   }
 
   return true;
-};
+}
 
 }
 #endif// FILTERS_INCREMENT_HPP_
```

Comment:
I know you didn't touch this, but the header guard is wrong at the top of the file.  It *should* be `FILTERS_INCREMENT_HPP`, but it is actually `FILTERS_MEAN_HPP`.

Reply:
Fixed include guard in d17e99042e76684802a98d4b3574c6ecd98022f5